### PR TITLE
Telemetry:  Make Driver naming  consistent with exporter

### PR DESCRIPTION
**Pull request by NajmudheenCT**
**What this PR does / why we need it**:
1. Making changes to the names and fields related to forming prometheus gauge name to make the naming consistent with exporter ( starting all names with small letter)
2. Added exporter build in make file
3. Separated label generation out of metric spec filling logic.
4. Make file changes to build exporter.

**Special notes for your reviewer**:
Patches to harden the driver and exporter

### CODE REVIEW

File: contrib/drivers/lvm/lvm_metrics.go [hunk 1]
```diff
@@ -28,32 +28,34 @@ var data = `
...
   - resource: volume
     metrics:
-      - IOPS
-      - ReadThroughput
-      - WriteThroughput
-      - ResponseTime
-      - ServiceTime
-      - UtilizationPercentage
+      - iops
+      - readThroughput
```
Comment (xing-yang) [inline contrib/drivers/lvm/lvm_metrics.go]: read_throughput

Comment (NajmudheenCT) [reply contrib/drivers/lvm/lvm_metrics.go]: Fixed , thanks

**Comment by NajmudheenCT** [comment]
@skdwriting @rhsakarpos @wisererik @leonwanghui

**Comment by leonwanghui** [comment]
Please update your code.

**Comment by NajmudheenCT** [comment]
> Please update your code.

Done, Thanks

**Pull request merged**